### PR TITLE
Integration testing for assignment

### PR DESCRIPTION
**Pull request by akshatakanaje**

### CODE REVIEW

File: src/test/java/com/numpyninja/lms/controller/AssignmentControllerIT.java [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+
+@ExtendWith(MockitoExtension.class)
+@AutoConfigureDataJpa
+@WebMvcTest(UserController.class)
```
Comment (gkav01) [inline src/test/java/com/numpyninja/lms/controller/AssignmentControllerIT.java]: Change this to AssignmentController.class

**Review by gkav01 — APPROVED** [review]
Code looks fine, small changes suggested. Please make them and merge code

File: src/test/java/com/numpyninja/lms/controller/AssignmentControllerIT.java [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        assertEquals("FilePath1", responseDto .getPathAttachment1());
+        assertEquals("FilePath2", responseDto .getPathAttachment2());
+        assertEquals("FilePath3", responseDto .getPathAttachment3());
+        assertEquals("FilePath4", responseDto .getPathAttachment4());
+        assertEquals("FilePath5", responseDto .getPathAttachment5());
+        assertEquals("U01", responseDto .getCreatedBy());
+        assertEquals(1, responseDto .getBatchId());
+        assertEquals("U01", responseDto .getGraderId());
+
+        assignmentId = responseDto.getAssignmentId();
```
[the remark this answers is not in the document]
Comment (akshatakanaje) [reply src/test/java/com/numpyninja/lms/controller/AssignmentControllerIT.java:116]: will do that

File: src/test/java/com/numpyninja/lms/controller/AssignmentControllerIT.java (hunk 1 above)
Comment (akshatakanaje) [reply src/test/java/com/numpyninja/lms/controller/AssignmentControllerIT.java]: Yes, Thank you

**Pull request closed without merging**